extensions/wikia/Track/Track.php

```diff
@@ -65,6 +65,7 @@ private static function getViewJS ($param=null) {
 		window.beacon_id = result[1];
 	} else {
 		// something went terribly wrong
+		document.write('<img src="http://logs-01.loggly.com/inputs/88a88e56-77c6-49cc-af41-6f44f83fe7fe.gif?message=something%20went%20terribly%20wrong"/>');
```

Comment:
A. not something but wikia_beacon_id is missing - could we write better message for it?
B. why not log it to our infrastructure? and make it possible to query for it in kibana?